src/box/session.h

```diff
@@ -375,6 +375,69 @@ generic_session_fd(struct session *session);
 int64_t
 generic_session_sync(struct session *session);
 
+/** \cond public */
+
+/**
+ * Get the unique identifier (ID) for the current session.
+ *
+ * @retval session ID
+ */
+API_EXPORT int
+box_session_id(void);
+
+/**
+ * Check if session exists.
+ *
+ * @param sid session ID
+ * @retval true if sessions exists, false otherwise
+ */
+API_EXPORT bool
+box_session_exists(uint64_t sid);
+
+/**
+ * Get host address and port of the session peer (if it exists).
+ *
+ * @param sid session ID
+ * @retval peer's host and port on success, NULL on error.
+ */
+API_EXPORT const char *
+box_session_peer(uint64_t sid, char *dst, size_t size);
+
+/**
+ * Get user's name for current session.
```

Comment:
Please, add a comment that the returned string memory does not belong to the user and might be deleted on any operation on the users, or on any yield. It must be copied if the user wants to keep it.